packages/plugins/i18n/admin/src/index.ts

```diff
@@ -71,6 +75,19 @@ export default {
       return actions;
     });
 
+    contentManager.apis.addDocumentAction((actions: DocumentActionComponent[]) => {
+      const hasPublishAction = actions.find((action) => action.type === 'publish');
+      if (!hasPublishAction) {
+        return actions;
+      }
```

Comment:
Are you doing this to determine if we have D&P? :)

Reply:
yep

Reply:
i'd probably just add it, and check the schema options are return `null` if it's not enabled? we add all the EE parts and then return null regardless